### ISSUE TITLE
games won't play

### ISSUE THREAD

**Issue by ironspider2k** [LibreVR/Revive#1087]
every game i play with revive doesn't work. usually i start and then it immediately goes back to steamvr. sometimes i can get into the game..and back to steamvr. but the longest i have been able to get into a game was maybe 3mins. nothing plays. i have used the latest version. even went back to 1.2.2. i get the same results on everything. i have no issues playing vive games. am i missing something?

**Comment by enrekatr** [comment]
Can u type your hardware and software like os/steamvr version etc?

**Comment by ironspider2k** [comment]
windows 7 ultimate 64bit
nvidia geforce gto 980
steamvr version 1539100633
last version of revive i tried 1.51

**Comment by kensprad08** [comment]
I have the same issue.  The games will start up and work for a very short time before they are closed down, probably by oculus.  The only way I have found to get it to work, is to download and install an older revive version, 1.42, and make sure you set your steamvr beta version to "v1527117754 - 6/21/2018 hotfix" from the dropdown list in properties.  Unfortunately you lose your controller support in steamvr home, so you cannot launch the games from the revive dashboard inside vr, you have to take off the headset and run it from the desktop.  Then, once I'm done with playing oculus games, I go to steamvr properties and switch the beta opt-in back to none.

**Comment by jwnmiles** [comment]
Same issue here I used remedy kensprad08 posted and worked. Is there going to be an update for this????

**Comment by DaveBH** [comment]
I have the same issue as ironspider2k.  Took a break from VR for awhile, and upon my return none of my Oculus games run for longer than a couple of minutes before returning to Steam.  In Lone Echo, the left controller sends random and occasionally insistent haptic feedback while in the start menu.  All Steam games run fine, I have tried the following games in Revive:

Lone Echo
Obduction
Bullet Train
Wilson's Heart

Windows 7 Home Premium 64 bit
Intel i7 3770
Nvidia GTX 980 Ti
Steam package versions: 1541710380

I am not running VR beta
I have tried the following
                  disable Steam VR Home Beta
                  disable AVG antivirus
                  run from ReviveOverlay
                  uninstall, rollback to 1.22 and 1.42
                  uninstall, reinstall current build
                  restart computer
I have not tried kensprad08's solution yet, I will give it a go this weekend. 
Any help is greatly appreciated!

**Comment by roachmojo1** [comment]
I installed Revive for the first time a couple of days ago and have been having issues with every game I have tried as well. Some will play for a short time, others will fail after about 10 seconds, and always with the message "Fatal Error". Vive games work without issues. 

Windows 7 Enterprise 64 bit, current updates
Intel i7 3770K
GTX 1080 Ti
Steam version: 1541819448
Revive - 1.5.1.1

I am not running VR beta, but I have tried disabling AVG, run from ReviveOverlay, restart computer

Echo Arena - fatal error in 10 -30 seconds
Air Car - fatal error in 10 -30 seconds, but will sometimes run for several  minutes
Showdown - fatal error in 10 -30 seconds
Face your Fears - fatal error at main menu

Thanks for your help on this!

**Comment by jwnmiles** [comment]
I updated Oculus drivers to latest updated steam vr beta to latest. Left Revive at 1.4.2 and it works so it is the Revive 1.5.1.1 that is the problem.

**Comment by ironspider2k** [comment]
yeah, ive updated to the latest on all of them. i was able to get battle tactics running but, once i got the game going..i couldnt move around the map. some progress :+1:

**Comment by DaveBH** [comment]
Ok, I've tried to rollback to 1.4.2 with no luck (Lone Echo crashes in less than 10 seconds), and I've tried the newest update (1.6.0) with no luck.  The controllers in 1.6.0 seem much more solid, but Lone Echo still crashes within 30 seconds of play.  
Anyone have any ideas?  Thanks!

**Comment by DaveBH** [comment]
So, I noticed that Windows 7 seems to be a common element, so decided to upgrade to windows 10.
I've  installed the latest Oculus drivers, the most recent version of Revive, and updated my video drivers. Reboot, of course.  Now Lone Echo crashes during loading, and Obduction hangs almost immediately after menu selection.  
Motion smoothing? Oversampling?  It seems to be a controller issue, although games crash before play, so it's hard to tell.

**Comment by DaveBH** [comment]
Ok, success.  All Oculus games are now running.
Most (Chronos, Edge of Nowhere, Bullet Train) ran after I upgraded to Windows 10 and updated all drivers.  The two stubborn games (Lone Echo and Obduction) did not run until I turned off Motion Smoothing in Vive settings.  Now everything is back to normal, save a small controller issue in Lone Echo (no longer able to get the index finger in pointing mode when pulling the trigger).
Hope this helps!
